### DISCUSSION TITLE
Follow system light/dark theme

### DISCUSSION THREAD

**Discussion by Nogooduser** [Arcticons-Team/Arcticons#1554] [Ideas]
Recently some icon packs have been popping up that can follow the system theme to switch between a light and dark version ([example 1](https://play.google.com/store/apps/details?id=com.jndapp.adaptive.material.you.iconpack), [example 2](https://play.google.com/store/apps/details?id=com.pashapuma.pix.material.you.dark)).

It would be really great if arcticons could have an apk version that can do the same (in addition to the already existing light and dark versions).

Use cases for this are people like myself that like changing wallpapers automacially based on the system theme. Having an icon pack that can automatically change along with the wallpaper when the device goes in and out of dark mode is very handy.

**Comment by Donnnno** [comment]
Hey, this is a really cool idea, and many had it before haha. 
Previously, it wasn't possible to adjust them from the icon pack, it had to be modified on the launchers side, but I'm curious how other packs have done it.

These issues are probably related to this request:
https://github.com/Donnnno/Arcticons/issues/404
https://github.com/Donnnno/Arcticons/issues/179
https://github.com/Donnnno/Arcticons/issues/37
https://github.com/Donnnno/Arcticons/issues/16

**Comment by Donnnno** [comment]
Hi @Nogooduser ! We've just released the Day & Night version here on GitHub, check it out!
